e2e/single-cluster/helm_auth_test.go

```diff
@@ -62,9 +64,38 @@ var _ = Describe("Single Cluster Examples", Label("infra-setup"), func() {
 				}).Should(ContainSubstring("sleeper-"))
 			})
 		})
+		Context("containing a private HTTP-based helm chart with repo path and no configured Helm secret name", Label("helm-registry"), func() {
```

Comment:
It says "and no configured Helm secret"  but it is passing a secret? is it a typo?

Reply:
Good catch! Fixed, thanks :)